jobs/cloud_controller_ng/templates/scrape.crt.erb

```diff
@@ -0,0 +1,4 @@
+<% if_p('cc.prom_scraper_tls.public_cert') do %>
```

Comment:
A shorter syntax would be:
```
<%= p('cc.prom_scraper_tls.public_cert', '') %>
```
I.e. add cert if provided, otherwise an empty string.

The same could be done for the private key and the CA cert.

Reply:
Changed to the shorter syntax